lib/document.js

```diff
@@ -2218,7 +2218,9 @@ Document.prototype.isSelected = function isSelected(path) {
   if (this.$__.selected == null) {
     return true;
   }
-
+  if (!path) {
```

Comment:
This is a coarse solution. This check is helpful, but you should also add a check in `$__version()` to avoid calling `isSelected()` if `key === false`